checkbox-support/checkbox_support/parsers/udevadm.py

```diff
@@ -561,9 +561,14 @@ def category(self):
             if self._environment["SUBSYSTEM"] == "hidraw":
                 return "HIDRAW"
             if self._environment["SUBSYSTEM"] == "video4linux":
-                if self.driver not in ('bcm2835-codec', 'bcm2835-isp'):
-                    # Ignore raspberrypi memory to memory (M2M) devices,
-                    # for the video decoder, encoder, and ISP resize.
+                # Ignore raspberrypi memory to memory (M2M) devices,
+                # for the video decoder, encoder, and ISP resize.
+                # Ignore Mediatek v4l2 encoder, decoder, jpeg codec and
+                # image processor 3
+                driver_black_list = ('bcm2835-codec', 'bcm2835-isp',
```

Comment:
Can you use `drivers_blocklist` for this variable instead?

Reply:
Hi @pieqq, I updated it. Please review it again. Thanks